internal/controller/cluster_monitoring_controller.go

```diff
@@ -124,6 +127,12 @@ func (r *ClusterMonitoringReconciler) reconcile(ctx context.Context, cluster *cl
 		return ctrl.Result{RequeueAfter: 5 * time.Minute}, errors.WithStack(err)
 	}
 
+	err = r.MimirService.ReconcileMimirConfig(ctx, r.ManagementCluster.Name)
```

Comment:
```suggestion
	err = r.MimirService.ConfigureMimir(ctx, r.ManagementCluster.Name)
```
?

Reply:
Btw, I would prefer if you add the management cluster in the service instead of here. WDYT?

Reply:
I'd still keep the `reconcile` in the name because that's basically what it does

Reply:
That's what thé controller does, not the service :)

Reply:
You have a point ! Updated it